src/test/java/com/yana/internship/service/ApartmentServiceTest.java

```diff
@@ -0,0 +1,101 @@
+package com.yana.internship.service;
+
+import com.yana.internship.entity.Address;
+import com.yana.internship.entity.Apartment;
+import com.yana.internship.entity.Image;
+import com.yana.internship.entity.Tariff;
+import com.yana.internship.repository.AddressRepository;
+import com.yana.internship.repository.ApartmentRepository;
+import com.yana.internship.repository.ImageRepository;
+import com.yana.internship.repository.TariffRepository;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Spy;
+import org.mockito.junit.MockitoJUnitRunner;
+import org.springframework.data.jpa.domain.Specification;
+
+import java.util.List;
+import java.util.Optional;
+
+import static org.junit.Assert.assertEquals;
+import static org.mockito.Mockito.*;
+
+@RunWith(MockitoJUnitRunner.class)
+public class ApartmentServiceTest {
+
+    @Spy
+    @InjectMocks
+    ApartmentService apartmentService;
+
+    @Mock
+    ApartmentRepository apartmentRepository;
+    @Mock
+    TariffRepository tariffRepository;
+    @Mock
+    AddressRepository addressRepository;
+    @Mock
+    ImageRepository imageRepository;
+
+    @Test
+    public void createShouldReturnCorrectResult() {
+        Apartment apartment = createApartment();
+        when(apartmentRepository.save(apartment)).thenReturn(apartment);
+        assertEquals(apartment, apartmentService.create(apartment));
+    }
+
+    @Test
+    public void createShouldCallRepository() {
```

Comment:
Please verify all repositories of the method.
Also would be good to validate value which go to the method `save`